packages/shared/notionable-hooks/src/use-liquidation-prices.tsx

```diff
@@ -138,7 +138,10 @@ function parseUnderlyingLiquidationPrice(
   return {
     // Used on portfolio screen
     exchangeRate: {
-      symbol: icon,
+      symbol:
+        asset.tokenType === 'VaultShare' && threshold?.toUnderlying()?.symbol
```

Comment:
can you refactor this to just
`asset.tokenType === 'VaultShare' && threshold ? ...`

That will save a call to `toUnderlying()` which does take some time